README.md

```diff
@@ -143,6 +143,17 @@ runner.run(collection, {
                 agentOptions: { keepAlive: true, timeout: 399 }
             },
             https: new https.Agent({ keepAlive: true })
+        },
+
+        // helper to refresh oauth2 tokens during execution
+        refreshTokenHelper: {
+            timeout: 30000, // timeout for the refresh token request in ms, defaults to 30000 ms
+            refreshToken: async function (authId) {
```

Comment:
Since everywhere else we're using callbacks, should we keep things consistent by using callbacks? Or do we want to transition towards promises? @codenirvana